<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSPMainDialog</class>
 <widget class="QWidget" name="OSPMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>461</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="leftSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>Open Sky Planetarium</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="rightSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidg">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="calTab">
      <attribute name="title">
       <string>Calibration</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QGroupBox" name="laserDirControl">
           <property name="toolTip">
            <string/>
           </property>
           <property name="title">
            <string>LASER Direction Control</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="2">
             <widget class="QPushButton" name="mvDown">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Makes the LASER move downward only if it is not at (0,0).</string>
              </property>
              <property name="text">
               <string>down</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="mvUp">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Makes the LASER move upwards upto 90 degrees from (0,0).</string>
              </property>
              <property name="text">
               <string>up</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="mvRight">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Makes the LASER move rightwards upto 360 degrees from (0,0).</string>
              </property>
              <property name="text">
               <string>right</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="mvLeft">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Makes the LASER move leftwards only if it is not at (0,0).</string>
              </property>
              <property name="text">
               <string>left</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="4">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="reset">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Takes the LASER to (0,0).</string>
              </property>
              <property name="text">
               <string>reset</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="MotorSwitchingControl">
           <property name="title">
            <string>Motor Speed Control</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>46</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="CoarseAdj">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="toolTip">
               <string>Makes the movement of LASER fast during its direction control.</string>
              </property>
              <property name="text">
               <string>Coarse</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="FineAdj">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Makes the movement of LASER slow during its direction control. </string>
              </property>
              <property name="text">
               <string>Fine</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>45</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="laserlightControl">
           <property name="contextMenuPolicy">
            <enum>Qt::PreventContextMenu</enum>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="title">
            <string>LASER  Light Control</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="laserOn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="toolTip">
               <string>Turns the LASER on and enables slider to control intensity.</string>
              </property>
              <property name="text">
               <string>Turn ON</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="laserOff">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Turns the LASER off and sends the slider to its initial position. </string>
              </property>
              <property name="text">
               <string>Turn OFF</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="1">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QSlider" name="intensity">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Controls the intensity of LASER. Drag the slider to change the intensity.</string>
              </property>
              <property name="minimum">
               <number>50</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="sliderPosition">
               <number>50</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="configDevice">
           <property name="title">
            <string>Device Configuration</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="selectDev">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Selects the connected device.</string>
              </property>
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="startCal">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Initializes calibration using 2 or 3 reference stars. Also, it sets the current position of LASER as (0,0).</string>
              </property>
              <property name="text">
               <string>Start Calibrate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="goTo">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Takes the LASER to selected star after calibration has been accomplished.</string>
              </property>
              <property name="text">
               <string>GoTo</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="refStatus">
           <property name="title">
            <string>Reference Status</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="setRef">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Sets reference when a star is selected. It is disabled when three references have been set successfully.</string>
              </property>
              <property name="text">
               <string>Set Ref.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="refStat">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Displays the number of references set.</string>
              </property>
              <property name="text">
               <string>0/3</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="title">
            <string>Current Coordinates</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="toolTip">
                 <string>Displays the current horizontal position of LASER in degrees. </string>
                </property>
                <property name="text">
                 <string>X:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="X">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="toolTip">
                 <string>Displays the current vertical position of LASER in degrees.</string>
                </property>
                <property name="text">
                 <string>Y:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="Y">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="scriptTab">
      <attribute name="title">
       <string>ScriptEngine</string>
      </attribute>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>341</x>
         <y>41</y>
         <width>101</width>
         <height>291</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSlider" name="volumeChanged">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>BusyCursor</cursorShape>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="playui">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopui">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
       <widget class="QPushButton" name="openScript">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveScript">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="compScript">
        <property name="text">
         <string>Build</string>
        </property>
       </widget>
       <widget class="QPushButton" name="execScript">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Build + Run</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>23</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="gt">
        <property name="text">
         <string>Goto</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pl">
        <property name="text">
         <string>Audio</string>
        </property>
       </widget>
       <widget class="QPushButton" name="wt">
        <property name="text">
         <string>WAIT</string>
        </property>
       </widget>
       <widget class="QPushButton" name="lo">
        <property name="text">
         <string>LASER ON/OFF</string>
        </property>
       </widget>
      </widget>
      <widget class="QTextEdit" name="scriptEdit">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>42</y>
         <width>321</width>
         <height>291</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhLowercaseOnly|Qt::ImhMultiLine|Qt::ImhNoAutoUppercase</set>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="abtTab">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:20pt; font-weight:600; text-decoration: underline;&quot;&gt;OpenSkyPlanetarium&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;An Open Source Initiative by IIT Bombay to bring interesting objects in the Open Sky closer to the common masses and school childrens.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;Contact: IIT Bombay&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;Email: openskyplanetarium@fossee.in&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
